routingpy/routers/graphhopper.py

```diff
@@ -388,9 +388,10 @@ def _parse_directions_json(response, algorithm, elevation):
         if algorithm == "alternative_route":
             routes = []
             for route in response["paths"]:
-                geometry = [
-                    list(reversed(coord)) for coord in utils.decode_polyline5(route["points"], elevation)
-                ]
+                geometry = utils.decode_polyline5(
+                    route["points"],
+                    elevation,
+                )
```

Comment:
why not here in `lnglat` order?

Reply:
Missed that one, thanks for pointing out!

Reply:
Could also change the default to "lnglat", so we wouldn't need to specify it

Reply:
true, good idea!